### PR TITLE
Merge protocVersion and protobufVersion

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,6 @@ subprojects {
         oracleDriverVersion = '19.8.0.0'
         sqlserverDriverVersion = '8.4.1.jre8'
         grpcVersion = '1.46.0'
-        protocVersion = '3.20.1'
```
Comment (brfrn169) [inline build.gradle:36]: Removed `protocVersion`.

File: rpc/build.gradle [hunk 2]
```diff
@@ -14,7 +14,7 @@ javadoc {
 }
 
 protobuf {
-    protoc { artifact = "com.google.protobuf:protoc:${protocVersion}" }
+    protoc { artifact = "com.google.protobuf:protoc:${protobufVersion}" }
```
Comment (brfrn169) [inline rpc/build.gradle:17]: Use `protobufVersion` instead of `protocVersion`.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**